_shared/project/Makefile

```diff
@@ -5,11 +5,11 @@ help = help::; @echo $$$$(tput bold)$(strip $(1)):$$$$(tput sgr0) $(strip $(2))
 $(call help,make help,print this help message)
 
 .PHONY: services
-{% if cookiecutter.get("services") == "yes" %}
+{% if cookiecutter.get("postgres") == "yes" or include_exists("docker-compose/services.yml") %}
 $(call help,make services,start the services that the app needs)
 services: args?=up -d
 services: python
-	@tox -qe dockercompose -- $(args)
+	@docker compose $(args)
```

Comment:
As of [Docker Desktop 4.12.0](https://docs.docker.com/desktop/release-notes/#4120) there's now a `docker compose` command that ships with Docker, so the `docker-compose` Python package is no longer needed and `make services` no longer needs to use tox.

Reply:
Yes, although we only need to do that when we apply the cookiecutter to projects, not right away when we move this PR. I think most of the time is on macOS where they've been using Docker Desktop since forever so this shouldn't break them anyway